canonical/canonical.com
# WD-2904 Add Kathy Pelich bio and updates a few large images

## Done

- Added [Kathy Pelich bio](https://docs.google.com/document/d/1AVu14TzwCdmAJCfC-xzkW0Q_ia3vafoRN5KfbKcbyeU/edit)
- Updated [Hanna Neuborn's bio](https://docs.google.com/document/d/1N3SkGpk43Wl7jZzM9WxbSctGF5J2xt9h8O9-t6zHeQ8/edit)
- Updated the extremely large images

## QA

- Check the copy matches the copy docs
- Copy the new base64 images into the browser URL and see the image renders and that it is the right person
